README.md

```diff
@@ -39,23 +39,41 @@ Generally, you'll want to build from the `master` branch, unless you're testing
+
+When running locally-hosted development builds, you can launch the server in "unsafe" mode (by specifying the `unsafe` program argument). This will disable the check that would normally prevent the server from running while using the default secret. This allows you to fire up Phantazm instances for local testing without needing to configure a unique shared secret first.
+
+When launching Phantazm through IntelliJ (such as by using the `Run server` or `Run server + Velocity` run configurations), `unsafe` mode is **enabled**, so you don't have to do anything extra.
+
+**Warning**: Make sure your locally-hosted development server & proxy are not visible from the Internet if you're launching in unsafe mode!
+
+### For production
+
+If the `unsafe` argument is not specified, the server *will refuse to run* with the default secret, for security reasons. You'll need to configure your own secret.
+
+First, come up with an appropriate string. You can use a password manager, or a generation tool like `apg` or `pwgen`. If your server is accessible from the Internet, make sure it's suitably complex. Keep in mind that your secret isn't something you need to actually *remember*, and thus there's no real drawback to making it as long as you want.
+
+Next, set the `proxySecret` field in `./run/server-1/server-config.toml`, and the `forwarding-secret` field in `./run/velocity/velocity.toml`, to the same string (your secret).
 
 **Warning**: The secret is stored, in plaintext, in both the server and proxy configuration files. This is, regrettably, something we can't do much about. Make sure access to your backend servers is properly secured.
 
-Next, set the `velocitySecret` field in `./run/server-1/server-config.toml`, and the `forwarding-secret` field in `./run/velocity/velocity.toml`, to the same string (your secret).
+### Running
+
 
 The proxy will (by default) bind to `0.0.0.0:25565` and the server to `0.0.0.0:25567`, so you can connect through the proxy by adding the `localhost` server address in your Minecraft client.
 
-If you're using IntelliJ, you can launch this testing server using the `Run server` run configuration, or the server and proxy at once using the `Run server + Velocity` configuration.
+If you're using IntelliJ, you can launch this testing server using the `Run server` run configuration (the server will use unsafe mode), or the server and proxy at once using the `Run server + Velocity` configuration.
 
 You can also run the server and proxy from the command line as follows: \
-`java -jar ./run/server-1/server.jar` \
-`java -jar ./run/velocity/velocity.jar`
+`./gradlew setupServer copyJar` \
+`cd ./run/server-1` \
+`java -jar server.jar` \
+`cd ../velocity` \
+`java -jar .velocity.jar`
```

Comment:
technically these steps are not something you run in order since you need a terminal just to run the server jar...but idk if we need to be so specific lol

Reply:
i think people should be able to figure out that they have to cd to the root project dir first

Reply:
I mean once you do java -jar server.jar you can't just cd velocity

Reply:
just open a new tty :triumph: 
okay point taken